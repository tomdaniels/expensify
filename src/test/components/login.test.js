import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/login';

test('should render the login page correctly', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
});