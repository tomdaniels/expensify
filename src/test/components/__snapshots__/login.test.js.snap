// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the login page correctly 1`] = `
<div
  className="login-page__box-layout"
>
  <div
    className="login-page__box"
  >
    <h1
      className="login-page__title"
    >
      Expensify
    </h1>
    <p>
      It's time to get your expenses under control
    </p>
    <button
      className="button"
    >
      Login with Google
    </button>
  </div>
</div>
`;
